src/stateless/DefaultListItem.vue

```diff
@@ -20,8 +20,8 @@
 
         <div v-if="metadata" class="default-list-item__hidden-widths">
             <p ref="labelContainer" class="default-list-item__label">{{ label }}</p>
-            <p ref="metadataContainer" class="default-list-item__metadata">
-                {{ metadata }}
+            <p class="default-list-item__metadata">
+                <span ref="metadataContainer">{{ metadata }}</span>
```

Comment:
@zan-kusterle I think in order to fix `calculateWidths` we must first make this change. Only with this you get actual  `clientWidth`. But then to make this work in all cases there was so much other stuff that I just forked this Item component and make an ugly copy-paste for what we need for now.

Can a separate ListItem component be passed as a slot through `multiselect.vue`?